src/busesApp/services.js

```diff
@@ -25,7 +25,7 @@ export const updateBus = async (Model, busId, editInfo) => {
 	const bus = await Model.findOneBy({ id: busId });
 	if (!bus) return false;
 
-	await Model.update(bus, editInfo);
+	await Model.updateById(bus.id, editInfo);
```

Comment:
I have noticed that some of the edits are working but @samuel-shyaka-dus can you confirm that editing plate numbers is working as expected it seems to not work on my end

Reply:
There was an issue in the `updateValidation.js` file. Now it is fixed